firebase-inappmessaging-display/firebase-inappmessaging-display.gradle

```diff
@@ -15,11 +15,11 @@
 apply plugin: "com.android.library"
 
 android {
-    compileSdkVersion 27
+    compileSdkVersion 28
```

Comment:
Just to confirm, is this change (and below) intentional?

Reply:
Yep, robolectric 4+ requires minimum compileSdkVersion = 28